e2e/testdata/test-service.json

```diff
@@ -0,0 +1,85 @@
+{
+  "sid": "test-service",
+  "name": "test-service",
+  "configuration": {},
```

Comment:
It could be good to add a service with envs, volumes, dependencies, etc...

I'm using this service for my tests, I have volumes, volumesFrom, envs, dependencies, env override and test of all that when the service starts
```json
{
  "sid": "testservice",
  "name": "test service",
  "configuration": {
    "volumes": [
      "/volume/test/"
    ],
    "volumesFrom": [
      "nginx"
    ],
    "env": [
      "ENVA=do_not_override",
      "ENVB=override"
    ]
  },
  "dependencies": [
    {
      "image": "nginx",
      "volumes": [
        "/etc/nginx"
      ],
      "key": "nginx"
    }
  ],
  "tasks": [
    {
      "key": "taskx",
      "inputs": [
        {
          "key": "foo",
          "type": "String",
          "object": []
        },
        {
          "key": "bar",
          "type": "String",
          "object": []
        }
      ],
      "outputs": [
        {
          "key": "result",
          "type": "String",
          "object": []
        }
      ]
    }
  ],
  "events": [
    {
      "key": "event",
      "data": [
        {
          "key": "x",
          "type": "Any",
          "object": []
        }
      ]
    }
  ],
  "source": "QmbNWV7UV4hHj62SVjFEWyAroSBepgDMWg45x6CfkqbLfb"
}
```

Reply:
This could be the next step. For now on I just wanted to test real simple service.

Reply:
Created related issue https://github.com/mesg-foundation/engine/issues/1446